src/public/interfaces.ts

```diff
@@ -418,6 +418,11 @@ export interface OpenConversationRequest {
    */
   entityId: string;
 
+  /**
+   * The subtext of the tab
+   */
+  subtext?: string;
```

Comment:
How did we come up with the `subtext` name? I think it might be better to call this `subtitle` or `description`.

Reply:
Loki provided that in the work item. Will sync with him.